packages/mdc-ripple/README.md

```diff
@@ -16,34 +16,46 @@ MDC Ripple also works without JavaScript, where it gracefully degrades to a simp
 ## Table of Contents
 
 - [An aside regarding browser support](#an-aside-regarding-browser-support)
+- [Design & API Documentation](#design--api-documentation)
 - [Installation](#installation)
 - [Usage](#usage)
-  - [Adding Ripple styles](#adding-ripple-styles)
-  - [Adding Ripple JS](#adding-ripple-js)
-  - [Ripple JS API](#ripple-js-api)
-  - [Unbounded Ripples](#unbounded-ripples)
-  - [The mdc-ripple-surface class](#the-mdc-ripple-surface-class)
-  - [Using the foundation](#using-the-foundation)
-  - [Using the vanilla DOM adapter](#using-the-vanilla-dom-adapter)
+  - [CSS Classes](#css-classes)
+  - [Sass Mixins](#sass-mixins)
+  - [MDCRipple](#mdcripple)
+  - [MDCRippleAdapter](#mdcrippleadapter)
+  - [MDCRippleFoundation](#mdcripplefoundation)
 - [Tips/Tricks](#tipstricks)
-  - [Integrating ripples into MDC-Web components](#integrating-ripples-into-mdc-web-components)
   - [Using a sentinel element for a ripple](#using-a-sentinel-element-for-a-ripple)
+  - [Unbounded ripple](#unbounded-ripple)
+  - [MDCRipple with custom functionality](#mdcripple-with-custom-functionality)
   - [Keyboard interaction for custom UI components](#keyboard-interaction-for-custom-ui-components)
   - [Specifying known element dimensions](#specifying-known-element-dimensions)
-- [Caveat: Edge](#caveat-edge)
-- [Caveat: Safari 9](#caveat-safari)
-- [Caveat: Mobile Safari](#caveat-mobile-safari)
-- [Caveat: Theme Custom Variables](#caveat-theme-custom-variables)
-- [The util API](#the-util-api)
+  - [The util API](#the-util-api)
+- [Caveats](#caveats)
+  - [Caveat: Edge](#caveat-edge)
+  - [Caveat: Safari 9](#caveat-safari)
+  - [Caveat: Mobile Safari](#caveat-mobile-safari)
+  - [Caveat: Theme Custom Variables](#caveat-theme-custom-variables)
 
-### An aside regarding browser support
+#### An aside regarding browser support
```

Comment:
I dont think we need this aside at all. We already cover the CSS variable support with the caveats sections